source/develop/developer-guide/engine/hosted-engine-invalid-bond-modes.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Hosted-Engine invalid bond modes
+authors: Ido Rosenzwig
+---
+
+# Hosted-Engine invalid bond modes
+
+Some bond modes (0, 5, 6) cause problems in VM networks.
+Therefore, when the user has to select network device, during the Hosted-engine deployment,
+bonds that are configured in these modes cannot be selected.
+
+## Enable invalid bond modes
+
+If one wish to configure Hosted-Engine with unsupported bond mode, it can be done by doing the following:               
+ 
+1. create a config file e.g. /etc/ovirt-hosted-engine/my-config.conf with content:
+
+```
+[environment:default]
+OVEHOSTED_NETWORK/allowInvalidBondModes=bool:True
```

Comment:
I think I'd simply suggest to add above line to the answer file, or to pass '--otopi-environment'.

Perhaps we should introduce some conf.d directory for such files, so below won't be needed.

However, if you want to add a file without changing the code, you still have two other options:

1. Add it as /etc/otopi.conf.d/my-conf.conf
otopi defaults to /etc/otopi.conf, and ovirt-hosted-engine-setup does not override that.
This will break once we decide to override.

2. Add to env.d below:
OTOPI_CONFIG=/etc/my-stuff.conf

If you do this, it will break our future code that sets a different conf.

So best is to add code :-)